packages/extension/src/newtab/dnd.ts

```diff
@@ -15,9 +16,14 @@ export enum CustomTime {
 
 const DEFAULT_URL = 'https://www.google.com';
 const CHROME_DEFAULT_URL = 'chrome-search://local-ntp/local-ntp.html';
+const BRAVE_DEFAULT_URL = 'chrome://new-tab-page';
 
-export const getDefaultLink = (): string =>
-  process.env.TARGET_BROWSER === 'chrome' ? CHROME_DEFAULT_URL : DEFAULT_URL;
+export const getDefaultLink = (): string => {
+  const browserTest = () =>
+    process.env.TARGET_BROWSER === 'chrome' ? CHROME_DEFAULT_URL : DEFAULT_URL;
+  const braveTest = () => (isBrave() ? BRAVE_DEFAULT_URL : browserTest());
```

Comment:
Looks like we can take this out of the function.
Functions generating more functions inside usually utilize the parent function's scope but here it's not necessary :)